More Interactive and More Efficient !!

Devarsh-leo opened the pull request:
Modified File : linear-search.py
Folder: Algorithms in Python
New User Interface with more efficiency is added.
Menu is made to interact with user.
List comprehension is used instead of for loops to increase efficiency of the program.
Multiple values can searched or removed with only one time input.
While loop is placed so the program can be used until user wants to exit.
Previous code is still added in comments at the bottom of the file.

Devarsh-leo committed:
New User Interface with more efficiency is added
Menu is made to interact with user
list comprihension is used instead of for loops to increase efficiency of the
program
multiple values can searched or removed with only one time input
While loop is placed so the program can be used until user wants to exit
Previous code is still added in comments at the bottom of the file

Devarsh-leo committed:
File : linear-search.py
Folder: Algorithms in Python
New User Interface with more efficiency is added
Menu is made to interact with user
list comprihension is used instead of for loops to increase efficiency of the
program
multiple values can searched or removed with only one time input
While loop is placed so the program can be used until user wants to exit
Previous code is still added in comments at the bottom of the file

parthpandyappp commented:
Looks great!
Please rename your file, as `menu-driven-linear-search.py`
Thanks! Happy Hacking!

Devarsh-leo committed:
Merge branch 'main' of https://github.com/Devarsh-leo/Sorting-Searching-Algorithms into main

parthpandyappp commented:
@Devarsh-leo Do uncomment the previous code, and place your code in another file named `menu-driven-linear-search.py`

Devarsh-leo committed:
Added File : menu-driven-linear-search.py
Folder: Algorithms in Python
New User Interface with more efficiency is added.
Menu is made to interact with user.
List comprehension is used instead of for loops to increase efficiency of the program.
Multiple values can searched or removed with only one time input.
While loop is placed so the program can be used until user wants to exit.

parthpandyappp commented:
@Devarsh-leo Great work so far! 
Contributions merged!
Thanks, Happy Hacking!
